app/views/organizations/new.html.haml

```diff
@@ -0,0 +1,14 @@
+#main.ui.main.container
+  Create Organization
+  = form_for :organization, url: { action: 'create' }, html:{ class: "ui form" } do |f|
```

Comment:
This form is the same as the edit form, with a couple minor tweaks. Factor out the fields into a partial i.e. `organizations/_form`, and render the fields in the new and edit views.

Also, to avoid the need for `url: {action: 'create'}`, use `form_for @organization` instead of `form_for :organization`.